toptal/keycodes
# [SAT-2067] Remove service worker as it is not used anywhere

SAT-2067

### Description

Removed service worker file as it is not used anywhere

### How to test

- `npm ci && npm run dev`
- open `http://localhost:3000`
- the app should work as before :)

### Pre-merge checklist
